### ISSUE TITLE
Feature request: bring back the monster!

### ISSUE THREAD

**Issue by dromer** [clone45/voxglitch#241]
I know the new panels simplified things a lot, moving to full svg and standard components.
However the classic `Satanonaut` panel is really what made the voxglitch module collection come together somehow.

Please find a way to bring it back!

**Comment by clone45** [comment]
I know, and I totally agree.  I don't know how to make it happen yet, but I'll try. :-)

**Comment by CirKu17** [comment]
Yes please! The moster was the centerpiece of the collection and my personal muse. 
I miss it a lot.

**Comment by DrummerHead** [comment]
You could use something like this https://convertio.co/pix-svg/ to make the image into an SVG and then https://jakearchibald.github.io/svgomg/ to optimize the final SVG

Voxglitch was among the best visually designed modules in the whole VCV ecosystem :) sad to see the disappearance of grunge textures and the skeuomorphism that made it so cool

**Comment by clone45** [comment]
Ah!  Shoot!  I just lost a draft where I was saying that I totally agree!  I'll do what I can.  I'm trying to stay compatible with Cardinal, but I should be able to do that and use the strategies that you mentioned to make it happen.  A lot of the specialty knobs and widgets that Chis Corrado designed won't be possible, but the general design should be.  I'm thinking it might be possible to add an option to "satanify" the module (!?), which would reach out to a URL and pull in extended graphics for it?  Although, that wouldn't show up in the module browser, which would be unfortunate, because that was a lot of fun.  :-)

**Comment by dromer** [comment]
@clone45 it's not about compatibility necessarily. It's just that at some point your artwork files kind of blew up.
Making external http requests would _not_ be allowed in Cardinal for sure btw.

Having an iconic panel as part of the collection should imo be allowed (within reason of course). Trying to do an optimized svg like @DrummerHead suggested would be the first thing to try I think :)

**Comment by clone45** [comment]
> Making external http requests would not be allowed in Cardinal for sure btw

Yep, totally agreed!  Let me see what I can do.  Cardinal has some code that can be used to identify if it's being run in cardinal or not, which maybe I can use to rectify things and show a more detailed view.  It's not exactly that cardinal doesn't support it, but they're trying to make web based version, and it gets bogged down by the size of of artwork files.

I might be wrong, but I sense some discontent in your posting.  I want to acknowledge that I'm just an old dude trying to keep productive and happy by making modules!  I'll try my best to bring back the monster.  Kurt Burdick created the satanonaut artwork in one of his comics, and he was humbled by the outreach that y'all wanted the graphics to return!

![image](https://github.com/user-attachments/assets/aea66207-941a-43ac-b953-0ed6be5d9838)

**Comment by dromer** [comment]
> I might be wrong, but I sense some discontent in your posting.

Oh not at all! I really appreciated that you went out of your way to simplify the designs!

No worries and no hurries. We're all just enjoying your work as well :)

**Comment by clone45** [comment]
I totally appreciate that.  I felt especially bad about the satanonaut module.  I'll see what I can do.  As part of my apology, let me share something that I've been working on ... you're the first to see this!

![image](https://github.com/user-attachments/assets/fdd2a6b2-e3f3-4bc0-a625-c66321e2caed)

It's a sample playback module.  (Boring!)  But it will allow you to set markers.  Each marker will send a trigger output to one of 32 different outputs.  For example, you might set a marker to trigger an effect to affect the sample, or to trigger another sample.  I'm super excited about it!

**Comment by DrummerHead** [comment]
I see two options:

1. Use the "old school" already existing SVG over here: https://raw.githubusercontent.com/clone45/voxglitch/refs/tags/1.30.1/res/satanonaut_front_panel.svg
2. Find the PSD (most likely PSD for photoshop, but perhaps it was made on Gimp or Affinity Designer) file for https://github.com/clone45/voxglitch/blob/2.21.2/res/satanonaut/satanonaut_baseplate.png and then extract only the background image

I was planning to SVG the file in point 2. but it wouldn't make sense to SVG the embedded controls, as they will be handled in a different way. If you can provide me with the source file or just the image I can handle converting that into color SVG and optimizing it.

Damn, now this feels like we're part of some type of cult or something, trying to revive an ancient deity 👹

**Comment by clone45** [comment]
@DrummerHead An idea that I had recently was to create a blank panel with the Satanonaut artwork.  On the Satanonaut module, there would be an option to "Summon Satanonaut", which would automatically place the blank panel in the patch.  That would certainly be the easiest implementation?

**Comment by dromer** [comment]
For me the original panel was the best. The fully skeuomorphic colored one with weathered labels and connectors didn't hit in the same way :)

**Comment by clone45** [comment]
> For me the original panel was the best. The fully skeuomorphic colored one with weathered labels and connectors didn't hit in the same way :)

@dromer

You're in luck, because the original panel used SVG, and I'm likely to use that version!   :-)

**Comment by clone45** [comment]
I was able to pull down the SVG size to 215Kb, which seems acceptable to me.  Although, all of my other modules combined comes out to only a meg, but still, it's pretty small.

Some of the detail was obviously lost.  Here's a comparison between the compressed and original version.  The SVG is on the left.
![image](https://github.com/user-attachments/assets/2957b323-caa2-4091-a38d-1705cb5fb30d)

**Comment by DrummerHead** [comment]
I think the SVG version looks even better; great job!
